packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -119,6 +133,19 @@ export const getStyles = (props: ICheckboxStyleProps): ICheckboxStyles => {
       reversed && {
         flexDirection: 'row-reverse',
         justifyContent: 'flex-end'
+      },
+      {
+        selectors: {
+          '&::before': {

```

Comment:
should this be ::after?

Reply:
So, actually, I don't think the zIndex is necessary at all. I just included it to be consistent with `getFocusStyles` (which was what the code used to call). 

Everything still seems to work as far as I can tell if I remove it, but I'm not sure if there's some edge case I'm not considering. 

If zIndex isn't necessary, it would be better to be before so we don't even need the `pointerEvents: "none"`. ATM the focus rect is intercepting clicks onto the label e.g. to an embedded link because of the zIndex (and if it was `:after`).